### ISSUE TITLE
Colors of the Feedback tool

### ISSUE THREAD

**Issue by noraeisner** [zooniverse/Panoptes-Front-End#5332]
## Expected behavior

It would be great if a different combination of colors could be used for the feedback tool. At the moment it is red for false and green for true, however, red/green colour blindness is quite common. Can this be changed? @rogerhutchings @eatyourgreens ?

**Comment by eatyourgreens** [comment]
Looks like the colours are currently hardcoded in the feedback strategy code. It would be easy enough to change the values here: https://github.com/zooniverse/Panoptes-Front-End/blob/2f7c55797bc9837ae5f0e6407d7c28d78b7bbe9b/app/features/feedback/shared/strategies/drawing/column/feedback-mark.jsx#L4-L5

**Comment by srallen** [comment]
@beckyrother any thoughts on the colors to use? I think we should include a small guide in the feedback for whatever colors get used, red and green have cultural meaning for fail and success, so if we use something else, we'll need to communicate that.

**Comment by beckyrother** [comment]
From the survey it seems like it's just not clear what the feedback is indicating; maybe it would help to also include icons over the markings, similar to where the `X` appears while doing the task. As Sarah mentioned the red and green is nearly universal (with some exceptions) in meaning fail and success. You're right about the colorblindness issue but adding additional information rather than relying solely on color is a more effective way to ensure legibility.

**Comment by eatyourgreens** [comment]
@beckyrother I think this issue is referring to the PFE version of Planet Hunters: https://www.zooniverse.org/projects/nora-dot-eisner/planet-hunters-tess/classify

Here's a screenshot of the feedback in PFE: 
![A screenshot of feedback for a simulated transit in Planet Hunters Tess.](https://user-images.githubusercontent.com/59547/56680122-84d6cd80-66be-11e9-95e8-99015858cd61.png)
For that one, I wasn't really sure if it's telling me that I spotted a transit or that I missed one. I think I got the right place in the light curve, but the mark I drew was too broad, so red for a miss?

**Comment by beckyrother** [comment]
I think **icons and maybe a key** will help this. In doing this myself I also didn't draw exactly the right size so I got a red. What does the blue mean?

**Comment by eatyourgreens** [comment]
Icons would definitely make the feedback easier to understand. Something like a tick for success, cross for failure?

One of the drawbacks of the current implementation is that the colours are set in our code, but the feedback messages are written by the project owner. If we change the colours in the code, project owners have to go back into the project builder and update any mention of the colours in their text. Not a big deal, but it's an extra step where we could accidentally break a project without realising it.

I'm not sure what the blue is either. Are two translucent layers overlapping, and the blue is from the combined colours?

**Comment by eatyourgreens** [comment]
@beckyrother zooming in on my screenshot, I _think_ the blue is an allowed margin of error. So the red column is the actual transit and the blue column is the area that I should have drawn inside in order to pass the test.

**Comment by beckyrother** [comment]
After doing a couple classifications, that was my guess as well. @noraeisner is that correct? How do you feel about adding some iconography to the images to clear up some of this confusion?

**Comment by eatyourgreens** [comment]
On the new project, by the way, the feedback marks were generated randomly for each image, so the questions we got there were because the feedback says "you missed a transit" when there isn't a transit to miss, and that's confusing volunteers who've built up a lot of experience at spotting them.

In retrospect, we should have tried to use real data for the test.

**Comment by mcbouslog** [comment]
Sorry for the late comment, but can confirm in PFE the blue area is margin of error or tolerance as it's referred to in the code.

In the frontend rewrite lightcurve viewer the feedback doesn't show tolerance and targeted annotation differently, but as one green or red column.

Also noting, though unrelated to this specific issue, but I think important to share, because the lightcurve viewer (LCV) task records the annotation a little differently than the PFE column drawing task, the LCV feedback is similarly different. To be specific, in PFE the column x value is the start of the column, and the width proceeds right, whereas in the LCV the x value is the center of the column.

**Comment by lcjohnso** [comment]
I'm working on [a ticket](https://zooniverse.freshdesk.com/a/tickets/3672) where the research team is inquiring about making the color of success/failure regions in the feedback modal configurable.  Specifically, the red color is hard to see given the color of the subject image.

In addition to the objection to the current red/green color mapping due to color-blindness, there was a similar "blending into background" argument for configurable colors raised in PR https://github.com/zooniverse/Panoptes-Front-End/pull/5475.

I'm reopening this issue to argue in favor of being able to change the red/green default colors to another pair.  I acknowledge that the addition of iconography (green check / red X) would be a good accompanying addition.

**Comment by srallen** [comment]
I don't think we should give full ability for any color. Likely would give access to the same colors we allow for drawing tools in general: https://projects.invisionapp.com/dsm/zooniverse/primary-brand/folder/colors/5c68fab7fb26bb001855f29a

Would you check for and open a parallel issue if there isn't already one on the monorepo?

**Comment by lcjohnso** [comment]
Not sure there is a true parallel issue on the monorepo, but most relevant existing FEM issue concerns feedback implementation in general: https://github.com/zooniverse/front-end-monorepo/issues/1274

I agree that if colors were adjustable, choice could be constrained to one or two other choices of color pairs: add magenta+cyan as new color pair choice and black+white as new neutral color pair choice, accompanying red+green that remains default.  However, after follow-up with the project team I'm advocating for, I see selectable color pairs as secondary in importance to adding icons.

The team from the ticket I'm working on (Tabletop Dark Matter) was happy to use an icon-based solution and keep red/green colors fixed.  For their use case, the issue is less to do with the colors and more to do with the fact that their features are small in size -- see below.  Red failure markers are hard to see (blend in with darker background features) but no matter the color the markers are small and hard to see.
<img width="317" alt="feedback_improvement" src="https://user-images.githubusercontent.com/2006101/85292542-ae75bb00-b461-11ea-9575-4f3de9b91059.png">

Given this, I'd advocate for keeping single red/green color pair fixed and add either a green checkmark icon for success or red X icon for failure, to be shown next to feature.  @srallen -- If there's agreement on this solution, would it possible to implement this feature by early fall, say 15 September 2020?

**Comment by srallen** [comment]
@lcjohnso I don't think we'll have time to get this done before Sept 15.

**Comment by lcjohnso** [comment]
Not stale, stalebot!
